Xamarin.Forms.Platform.Android/Cells/SwitchCellRenderer.cs

```diff
@@ -58,6 +63,17 @@ void UpdateHeight()
 			_view.SetRenderHeight(Cell.RenderHeight);
 		}
 
+		void UpdateFlowDirection()
```

Comment:
This same UpdateFlowDirection code is repeated several times - seems like a candidate for an extension method.